### ISSUE TITLE
Daily meeting(8/12/2016)

### ISSUE THREAD

**Issue by Lai0723** [Lai0723/Online-Art-Gallery-Shopping#8]
Anything to report for today or  any problem?

**Comment by pqyyieeuyi** [comment]
I still doing the validation part and i just want to ask the master card and visa need to validate too?

**Comment by pqyyieeuyi** [comment]
Yup, maybe you can validate when user type the card number for visa card must begin with 4 , while the master card begin with 5.

**Comment by Lai0723** [comment]
Yup, maybe you can validate when user type the card number for visa card must begin with 4 , while the master card begin with 5.

**Comment by pqyyieeuyi** [comment]
oh, ok itry it now.

**Comment by Lai0723** [comment]
how about you kc?

**Comment by kcchan3012** [comment]
I almost done with the payment history table while i want to ask , what extra thing field need to include in the table?

**Comment by kcchan3012** [comment]
payment id, payment price and ?

**Comment by boonsenglim** [comment]
maybe payment type ( using master or visa card)?

**Comment by pqyyieeuyi** [comment]
purchased gallery name too

**Comment by Lai0723** [comment]
ya, i think that all for that table

**Comment by kcchan3012** [comment]
ok i add in now

**Comment by Lai0723** [comment]
how about boon seng?

**Comment by boonsenglim** [comment]
i just finished the gallery database and sent to you , can you link it?

**Comment by boonsenglim** [comment]
ya, i able to add the gallery into that cart and i just want to know how to display it?

**Comment by Lai0723** [comment]
i just finished the gallery database and sent to you , can you link it?

**Comment by boonsenglim** [comment]
ya, i able to add the gallery into that cart and i just want to know how to display it?

**Comment by pqyyieeuyi** [comment]
you can use grid view to display it.

**Comment by kcchan3012** [comment]
ya i agree with ben too.

**Comment by boonsenglim** [comment]
ok thanks guys !!!

**Comment by Lai0723** [comment]
So i think this all for today meeting , since all still developing the function so we will discuss tomorrow again .
